docs/workflows/oa_web.md

```diff
@@ -16,31 +16,56 @@ static folder, builds the website and then deploys it to Cloudflare.
+    │   ├── autocomplete.parquet: used for filtering in Cloudflare Worker.
+    │   ├── country: individual entity statistics files for countries. Used to build each country page.
+    │   │   ├── ALB.json
+    │   │   ├── ARE.json
+    │   │   └── ARG.json
+    │   ├── country.json: used to create the country table. First 18 countries used to build first page of country table
+    │   │                 and then this file is included in the public folder and downloaded by the client to enable the
+    │   │                 other pages of the table to be displayed. Copied into public/data folder.
+    │   ├── country.jsonl: used to generate the parquet file.
+    │   ├── country.parquet: to be used along with apache-arrow to enable filtering from a Cloudflare Worker.
+    │   ├── institution: individual entity statistics files for institutions. Used to build each institution page.
+    │   │   ├── 05ykr0121.json
+    │   │   ├── 05ym42410.json
+    │   │   └── 05ynxx418.json
+    │   ├── institution.json: used to create the institution table. First 18 institutions used to build first page of institution table
+    │   │                     and then this file is included in the public folder and downloaded by the client to enable the
+    │   │                     other pages of the table to be displayed. Copied into public/data folder.
+    │   ├── institution.jsonl: used to generate the parquet file.
+    │   ├── institution.parquet: to be used along with apache-arrow to enable filtering from a Cloudflare Worker.
+    │   └── stats.json: global statistics, e.g. the minimum and maximum date for the dataset, when it was last updated etc.
+    └── logos: country and institution logos. Copied into public/logos folder.
+        ├── country
+        │   ├── l: large logos displayed on country pages.
+        │   │   ├── ALB.svg
+        │   │   ├── ARE.svg
+        │   │   └── ARG.svg
+        │   └── s: small logos displayed in country table.
+        │       ├── ALB.svg
+        │       ├── ARE.svg
+        │       └── ARG.svg
+        └── institution
+            ├── l: large logos displayed on institution pages.
+            │   ├── 05ykr0121.jpg
+            │   ├── 05ym42410.jpg
+            │   └── 05ynxx418.jpg
+            └── s: small logos displayed in institution table.
+                ├── 05ykr0121.jpg
+                ├── 05ym42410.jpg
+                └── 05ynxx418.jpg
+```
 
-Zone Resources:
-* Include, Specific zone: <select the domain you will deploy to>
+## Airflow Variables
+In the config.yaml file, the following Airflow variables are required.
 
-```yaml
-cloudflare_api_token: http://:<cloudflare_api_token>@
-```
+### oa_web_data_bucket
+The name of the Google Cloud Storage bucket where the data will be uploaded.
 
-## External references
-* [Cloudflare Creating API tokens](https://developers.cloudflare.com/api/tokens/create)
+When you create the bucket, enable object versioning to keep previous releases.
```

Comment:
It would be good to add info on the Github token Airflow Conn as well, does this require any specific permissions?

Reply:
Yeah good point, I've added some instructions and will share the current token with you as it is already created.